### PR TITLE
Lt 1634 end users on open applications

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: conf/constants.py [hunk 1]
```diff
@@ -151,3 +151,6 @@ class CaseType:
     EXHIBITION = "exhibition_clearance"
     F680 = "f680_clearance"
     HMRC = "hmrc"
+
+
+MILITARY = "military"
```
Comment (iichr) [inline conf/constants.py:156]: Might be better to replicate the structure in api seeing as we now switch on these:
```
class GoodsTypeCategory:
    MILITARY = "military"
    CRYPTOGRAPHIC = "cryptographic"
    MEDIA = "media"
    UK_CONTINENTAL_SHELF = "uk_continental_shelf"
    DEALER = "dealer"
```

File: templates/case/slices/open-app-parties.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+					<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">{{ destination.name }}</td>
+					<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">{{ destination|get_address }}</td>
+					<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">{{ destination.sub_type.value }}</td>
+					<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">{{ destination.website|linkify }}</td>
+					<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">
+						{% include 'includes/flags.html' with flags=destination.flags list=True %}
+					</td>
+				</tr>
+			{% endfor %}
+			</tbody>
```
Comment (iichr) [inline templates/case/slices/open-app-parties.html:27]: indent one further?

Comment (stephanieboyd) [reply templates/case/slices/open-app-parties.html:27]: done

File: conf/constants.py (hunk 1 above)
Comment (stephanieboyd) [reply conf/constants.py]: cool, done

**Pull request merged**